src/aframe-streetmix-parsers.js

```diff
@@ -631,6 +631,8 @@ function processSegments (segments, showStriping, length) {
     } else if (segments[i].type === 'outdoor-dining') {
       groundMixinId = (variantList[1] === 'road') ? 'drive-lane' : 'sidewalk';
       segmentParentEl.append(createOutdoorDining(variantList, positionX));
+    } else if (segments[i].type === 'parklet'){
+      console.log("parklet");
```

Comment:
Nice work @mharegot :+1: As we discussed earlier today, we will pick up where we left off and implement a function that adds a parklet to the scene.